src/Kdyby/Monolog/Diagnostics/MonologAdapter.php

```diff
@@ -46,6 +51,15 @@ public function __construct(Monolog\Logger $monolog)
 
 		$this->directory = &Debugger::$logDirectory;
 		$this->email = &Debugger::$email;
+
+		$this->priorityMap = array(
```

Comment:
Sure. I am used to assign values to properties in the constructor. Shoud I move it then, I suppose?